gapii/cc/connection_header.cpp

```diff
@@ -60,8 +59,7 @@ bool ConnectionHeader::read(core::StreamReader* reader) {
   if (!reader->read(mObserveFrameFrequency) ||
```

Comment:
Should this be bumping the version to 3?

Reply:
since you are changing the size of the header, yes we should change the version number.

Reply:
Fixed.